src/index.js

```diff
@@ -0,0 +1,151 @@
+    landscape.textContent = land;
+};
+
+const tempIncrease = () => {
+    state.liveTemp += 1;
+};
+
+const tempDecrease = () => {
+    state.liveTemp -= 1;
+};
+changeTempAndLand();
+
+const displayCityInput = () => {
+    let cityInput = document.getElementById("city_input").value;
+    let cityDisplay = document.querySelector("h2");
+    cityDisplay.textContent = `For the city of  ${cityInput}`;
+}
+
+const getCityInput = () => {
+    let cityInput = document.getElementById("city_input").value;
+    return cityInput;
+}
+const tempConverter = (kelvin) => {
+    return (kelvin - 273.15) * 1.8 + 32;
+
+}
+const getWeatherInfo = (lat, lon) => {
+    const weatherRoute = API + weatherURL;
+    axios
+        .get(weatherRoute, { params: { lat : `${lat}`, lon : `${lon}`, format: "json", units: "imperial"} })
+        .then((result) => {
+            tempInKel = result.data["main"]["temp"];
+            let resultFahr = Math.round(tempConverter(tempInKel));
+            let cityTemp = document.getElementById("live_temp");
+            cityTemp.innerHTML = resultFahr;
+            state.liveTemp = resultFahr;
+            changeTempAndLand();
+        })
+};
+const getCityLiveWeather = () => {
+    const cityName = document.getElementById("city_input").value;
+    const locationRoute = API + locationURL;
+    let tempInKel = 0;
+    axios
+        .get(locationRoute, { params: { q : `${cityName}`, format: "json"} })
+        .then((result) => {
+            let coorLat = result.data[0]["lat"];
+            let coorLon = result.data[0]["lon"];
+            getWeatherInfo(coorLat, coorLon);
+        })
+}
+
+const resetCity = () => {
+    let cityDisplay = document.querySelector("h2");
+    cityDisplay.textContent = `For the city of  ${state.city}`;
+
+    let cityInput = document.getElementById("city_input");
+    cityInput.value = state.city;
+    getWeatherInfo(state.lat, state.lon); 
+}
+
+const selectSky = () => {
+    let skyPref = document.getElementById("sky").value;
+    let skyLine = ''
+    let skyColor = ''
+    if (skyPref === "sunny")  {
+        skyLine = "🕶️☀️ ☁️ ☀️ ☀️ ☁️ ☀️🕶️";
+        skyColor = "sunny";
+    } else if (skyPref === "cloudy") {
+        skyLine = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+        skyColor = "cloudy";
+    } else if (skyPref === "rainy")  {
+        skyLine = "🌧🌈🌧🌧💧🌧🌦🌧💧🌧🌧";
+        skyColor = "rainy";
+    } else if (skyPref === "snowy"){
+        skyLine = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+        skyColor = "snowy";
+    }
+    const paintSky = document.getElementById('sky_line');
+    paintSky.textContent = skyLine;
+    const skyBackground = document.querySelector(".garden_box");
+    console.log(skyBackground)
+    skyBackground.id = skyColor;
+}
+
+const registerEventHandlers = () => {
+    const tempIncButton = document.querySelector("#increase_button");
+    tempIncButton.addEventListener('click', tempIncrease);
+    tempIncButton.addEventListener('click', () =>{
```

Comment:
Instead of attaching two event listeners, you can call `changeTempAndLand()` inside the `tempIncrease()` function. Same goes for the `tempDecButton` below.